pkg/kubelet/kubelet_test.go

```diff
@@ -2176,17 +2176,12 @@ func TestPortForward(t *testing.T) {
 	var port uint16 = 5000
 	stream := &fakeReadWriteCloser{}
 
-	podInfraContainerImage := "POD"
 	infraContainerID := "infra"
-	// TODO: Move this test to dockertools so that we don't have to do the hacky
-	// type assertion here.
-	dm := kubelet.containerRuntime.(*dockertools.DockerManager)
-	dm.PodInfraContainerImage = podInfraContainerImage
 
 	fakeDocker.ContainerList = []docker.APIContainers{
 		{
 			ID:    infraContainerID,
-			Names: []string{"/k8s_" + podInfraContainerImage + "_" + podName + "_" + podNamespace + "_12345678_42"},
+			Names: []string{"/k8s_POD" + "_" + podName + "_" + podNamespace + "_12345678_42"},
```

Comment:
The cleanup is nice!

I wanted to move this test to dockertools/manager_test.go since it is one of the two tests in this file that still use fakeDocker (see #9081), but didn't have time to get to it.